dota-enums.d.ts

```diff
@@ -1,5 +1,6 @@
 /** @compileMembersOnly */
 declare enum FindType_t {
+    FIND_UNITS_EVERYWHERE = -1,
```

Comment:
Isn't FIND_UNITS_EVERYWHERE supposed to go into the radius field of FindUnitsInRadius? I think adding it to FindType_t is incorrect. If you want to use it you could just declare it separately I guess: `declare var FIND_UNITS_EVERYWHERE: number;`